cmd/gazelle/BUILD.bazel

```diff
@@ -34,14 +34,10 @@ go_library(
 go_binary(
     name = "gazelle",
     embed = [":go_default_library"],
-    visibility = ["//visibility:public"],
-)
-
-go_binary(
-    name = "gazelle_pure",
-    embed = [":go_default_library"],
+    msan = "off",
```

Comment:
This attribute is not documented btw.

Reply:
Thanks for pointing that out. Fixed in #1624.